universal-application-tool-0.0.1/conf/evolutions/default/3.sql

```diff
@@ -0,0 +1,33 @@
+# --- JSON-based tables
+
+# --- !Ups
+create table applicants (
+  id bigserial primary key,
+  idcs_id varchar,
```

Comment:
Let's leave this off until we need it. We may want to give it a more generic name like "sso_id" or something.
```suggestion
```